autoendpoint/src/settings.rs

```diff
@@ -31,6 +32,17 @@ pub struct Settings {
 
     pub vapid_aud: Vec<String>,
 
+    /// A stringified JSON list of VAPID public keys which should be tracked internally.
+    /// This should ONLY include Mozilla generated and consumed messages (e.g. "SendToTab", etc.)
+    /// These keys should be specified in stripped, b64encoded, X962 format (e.g. a single line of
+    /// base64 encoded data without padding).
+    /// You can use `scripts/convert_pem_to_x962.py` to easily convert EC Public keys stored in
+    /// PEM format into appropriate x962 format.
+    pub tracking_keys: String,
+    /// Cached, parsed tracking keys.
+    //TODO: convert to decoded Vec<u8>?
```

Comment:
It kinda is. 
Ideally, the key we track should be the bytearray of the decoded public key. This would prevent things like case confusion or formatting errors. Since we are using a consistently formatted, well known key, we can short cut things for now and just do string matches, but that's very much a cheat.

Reply:
Ok, but I think we can still kill this `tracking_vapid_pubs` field entirely since we're only calling `tracking_keys()` once at startup now. Then the comment should probably move to that method